.gitignore

```diff
@@ -40,6 +40,7 @@ rsconnect/
 **.nc.ind
 **_tasks.yml
 6_drivers/log/feather/*.feather.ind
+build/status/MV9ubGRhc19jdWjlcy9sb2cvTkxEQVNfM*.yml
```

Comment:
does this seem to be working?

Reply:
I don't think so...but I have a lot a lot a lot of these build yamls in here, and they have some pattern to them, but too many combinations to fight the battle against them this way even it it worked

Reply:
Yeah, OK...someday we'll figure it out, but maybe not this go-round.